core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardList.kt

```diff
@@ -53,8 +51,6 @@ fun <T> LazyListScope.newsResourceCardItems(
 
         NewsResourceCardExpanded(
             newsResource = newsResource,
-            isBookmarked = isBookmarkedMapper(item),
```

Comment:
we should keep `isBookmarked` and `onToggleBookmark` params. these are upcoming features in progress